BingBertSquad/nvidia_run_squad_deepspeed.py

```diff
@@ -877,23 +847,73 @@ def main():
     param_optimizer = [n for n in param_optimizer if 'pooler' not in n[0]]
 
     no_decay = ['bias', 'LayerNorm.bias', 'LayerNorm.weight']
+    no_freeze = ['qa_outputs']
     optimizer_grouped_parameters = [{
         'params':
-        [p for n, p in param_optimizer if not any(nd in n for nd in no_decay)],
+        [p for n, p in param_optimizer if not any(nd in n for nd in no_freeze)],
         'weight_decay':
-        0.01
-    }, {
+        0.00
+    },{
         'params':
-        [p for n, p in param_optimizer if any(nd in n for nd in no_decay)],
+        [p for n, p in param_optimizer if any(nd in n for nd in no_freeze)],
         'weight_decay':
-        0.0
+        0.0,
+        'non_freeze': False
     }]
```

Comment:
@tanghl1994 can we chat about this? we don't understand why we need this for 1bit-adam and also are concerned this will impact this code when running without 1bit-adam. Also we need to understand if other models that use 1bit-adam would need to do something similar here? /cc @awan-10